libs/SalesforceSDK/AndroidManifest.xml

```diff
@@ -116,16 +116,5 @@
                 <action android:name="com.google.firebase.MESSAGING_EVENT" />
             </intent-filter>
         </service>
-
-        <receiver android:name="com.salesforce.androidsdk.push.PushService$SFDCRegistrationRetryAlarmReceiver"
```

Comment:
All three of these declarations are no longer needed.  The alarm receiver was used by push service for the six days later registration retry, but now that's just a delay parameter when the work request is enqueued.

The services are also no longer needed since we're using Android background tasks.